index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="/src/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Inter"
+    />
+  </head>
+  <body>
+    <section class="first-page">
+      <header class="header">
+        <nav class="main-nav">
+          <ul class="nav-links">
+            <li class="nav-list">
+              <a class="nav-link" href="#">Portfolio</a>
+            </li>
+            <li class="nav-list"><a class="nav-link" href="#">About</a></li>
+            <li class="nav-list"><a class="nav-link" href="#">Contact</a></li>
+          </ul>
+        </nav>
+      </header>
+      <section class="headline">
+        <h2 class="title row1">Hey there. I’m Nicolae.</h2>
+        <h2 class="title row2">I'm a software developer</h2>
+        <p class="parag">
+          I can help you build a product, feature or website. Look through some
+          of my work and<br />experience! If you like what you see and have a
+          project you need coded, don’t hestiate<br />to contact me.
+        </p>
+        <div class="social-media">
+          <a
```

Comment:
- [ ] **REQUIRED:** It will be nice to group the list of icons uses in a ul tag with a li tag for each icon